src/routes/blog/post/turbopack-support-appwrite-sites/+page.markdoc

```diff
@@ -0,0 +1,48 @@
+---
+layout: post
+title: Announcing Turbopack support for Appwrite Sites
+description: Appwrite Sites now supports Next.js applications built with Turbopack, with faster builds and better compatibility for your deployments.
+date: 2025-09-16
+cover: /images/blog/turbopack-support-appwrite-sites/cover.png
+timeToRead: 3
+author: ebenezer-don
+category: announcement
+---
+
+Appwrite Sites now supports Next.js applications built with Turbopack, Vercel's bundler. This update fixes build compatibility issues and makes your Next.js deployments faster.
+
+# Why Turbopack matters
+
+Turbopack is built in Rust to fix webpack's performance issues. It delivers faster updates, up to 700x faster than webpack and up to 10x faster than Vite for large applications.
+
+Previously, deploying Next.js applications that used Turbopack on Appwrite Sites required manual intervention. Developers had to remove the `--turbo` flag from their build commands, forcing their applications back to webpack. This created friction for teams migrating from Vercel where Turbopack worked seamlessly.
```

Comment:
Is "from vercel" intentional tone here? should we be more generic?

Reply:
I mentioned it cause it's Vercel's bundler. I've removed it now though